jwst/pipeline/engdblog.cfg

```diff
@@ -0,0 +1,2 @@
+name = "engdblog"
+class = "jwst.engdblog.EngDBLogStep"
```

Comment:
That's what I thought should be, but I had looked through a number of them, and found that many did not have anything, so I went with less. Will put them in.

Also, wasn't sure what stspec would give, but since that is [failing at the moment](#649), could not find out. I'm actually going to take that up momentarily.